ui/app/styles/utils/_spacing.scss

```diff
@@ -4,6 +4,7 @@ $spacing-xxs: 4px;
 $spacing-xs: 8px;
 $spacing-s: 12px;
 $spacing-m: 16px;
+$spacing-xm: 18px;
```

Comment:
Since this value is only used in charts.scss I'd rather define it there and use it within that file's scope only, instead of mucking up our already-in-flux util spacing. Or maybe define here but as `$spacing-chart`